scripts/summit_bsub_openmc_nekrs.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -ne 3 ]; then
+  echo "usage: $0 <casename> <number of compute nodes> <hh:mm>"
+  exit 1
+fi
+
+if [ ! -d "$NEKRS_HOME" ]; then
+  echo "ERROR: The env variable NEKRS_HOME does not point to an existing directory"
+  exit 1
+fi
+
+if [ -z "$PROJ_ID" ]; then
+  echo "ERROR: The env variable PROJ_ID is not set"
+  exit 1
+fi
+
+: ${OCCA_CACHE_DIR:="$PWD/.cache/occa"}
+NVME_HOME="/mnt/bb/$USER/"
+XL_HOME="/sw/summit/xl/16.1.1-3/xlC/16.1.1"
+
+export NEKRS_HOME
+export OCCA_CACHE_DIR
+export NEKRS_HYPRE_NUM_THREADS=1
+export OGS_MPI_SUPPORT=1
+export OCCA_CXX="$XL_HOME/bin/xlc" 
+export OCCA_CXXFLAGS="-O3 -qarch=pwr9 -qhot -DUSE_OCCA_MEM_BYTE_ALIGN=64" 
```

Comment:
Mostly a curiosity question, but why do we use `gcc` for the ENRICO build but `xlc` for the OCCA pieces? Is it a performance issue or do certain pieces truly _require_ certain compilers?

Reply:
`libnekrs` requires GCC.  I believe it's necessary for nekRS's particular usage of dynamically-loaded libraries.  So it's easiest to build ENRICO and the rest of the libraries (`libopenmc`, `libocca`, etc.) with GCC too.  Just to be clear, here xlc is used for the JIT-compiled OCCA kernels, not `libocca` itself.